TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/hardware/MotorGroup.java

```diff
@@ -65,7 +65,26 @@ public void resetEncoders() {
         }
     }
 
+    public int getPosition(){
+        int pos = 0;
+        for(DcMotor motor : motors){
+            pos += motor.getCurrentPosition();
+        }
+        pos /= motors.size();
+        return pos;
+    }
+
+    public int getTargetPosition(){
```

Comment:
I am not sure the purpose of getting the average target position other than ... we can ...?